packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts

```diff
@@ -271,30 +269,50 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
       throw new Error('undefined fee')
 
     // We have to round because coinselect library uses sats per byte which cant be decimals
-    const fast = String(Math.round(feeData.data.fast?.satsPerKiloByte / 1024))
-    const average = String(Math.round(feeData.data.average?.satsPerKiloByte / 1024))
-    const slow = String(Math.round(feeData.data.slow?.satsPerKiloByte / 1024))
+    const fastPerByte = String(Math.round(feeData.data.fast?.satsPerKiloByte / 1024))
+    const averagePerByte = String(Math.round(feeData.data.average?.satsPerKiloByte / 1024))
+    const slowPerByte = String(Math.round(feeData.data.slow?.satsPerKiloByte / 1024))
```

Comment:
Why do you cast these as String here and then re-cast them as Number later?

If you need the value in both formats, putting in a BigNumber allows you to call `toNumber` and `toString` on the same value.